crypto/threads_win.c

```diff
@@ -675,7 +729,6 @@ int CRYPTO_atomic_load_int(int *val, int *ret, CRYPTO_RWLOCK *lock)
 
     return 1;
 #else
-    /* On Windows, LONG is always the same size as int. */
```

Comment:
You should keep this comment. `LONG` is always the same size as int! But not `long`!

Reply:
@t8m 
Thanks! The comment you suggested fits well, considering the source of the size confusion:
- Win32 VC x86/x64: `LONG`, `long`, `long int` are 32 bit
- MingW-W64: `LONG`, `long`, `long int` are 32 bit
- cygwin64: `LONG` is 32 bit, `long`, `long int` are 64 bit